languagetool-language-modules/en/src/main/java/org/languagetool/language/English.java

```diff
@@ -187,6 +187,7 @@ public LanguageMaintainedState getMaintainedState() {
         new EmptyLineRule(messages, this),
         new LongSentenceRule(messages, userConfig, 40),
         new LongParagraphRule(messages, this, userConfig),
+        //new OpenNMTRule(),     // commented out because of #903
         new ParagraphRepeatBeginningRule(messages, this),
```

Comment:
Is this okay? @danielnaber

Reply:
I'm not sure I understand... does it matter when the code is commented out?

Reply:
Sorry. You are right. It doesn't matter. This line was commented out in one place and was not present in the other.